_layouts/people.html

```diff
@@ -370,4 +370,143 @@ <h2>Details</h2>
         {% assign ols-roles = ols-roles | remove_first: ' / ' %}
         {% include _includes/people.html username=username user=user ols-roles=ols-roles %}
     {% endfor %}
+  {% for entry in site.data.people %}
```

Comment:
All changes below are not needed

Reply:
I think the text editor did that. I'll change that...

Reply:
Woah. 😮 This looks like a badly-resolved merge conflict. I'll delete right away!